src/configure

```diff
@@ -896,6 +945,41 @@ fi
 # Most of the OS-specific steps below will append to kaldi.mk
 echo "Doing OS specific configurations ..."
 
+if $android ; then
```

Comment:
I don't have a mac or a windows machine to try it on, but I suspect it will work fine if you have the proper android toolchain available